src/opera/threading/node_executor.py

```diff
@@ -0,0 +1,59 @@
+from concurrent.futures import ThreadPoolExecutor, wait
+from opera.error import OperationError
+
+WORKER_PREFIX = 'Worker'
+
+
+class NodeExecutor(ThreadPoolExecutor):
+    def __init__(self, num_workers=None):
+        super().__init__(
+            max_workers=num_workers,
```

Comment:
Should the max value of workers be upper limited somehow? I don't know if there can be any execution problems if we have like 10000000000000000000000000000000000000000000 workers.

Reply:
Well this number defines the maximum number of workers, not the required one. So unless we have 10000000000000000000000000000000000000000000 independent nodes in the blueprtint we shpuld be fine. What I forgot to add it a check that -w parameter must be greater than 0.

Reply:
Yes true, having 10000000000000000000000000000000000000000000 independent nodes is like the odds of winning the lottery. Good that you remembered to add checks for 0.